### PR TITLE
lifted-gen.go add flag parse

### PR DESCRIPTION
**Pull request by tanberBro**
Signed-off-by: tanberBro <pengfei.song@daocloud.io>

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by RainbowMango** [comment]
Thanks @tanberBro for doing this. 
Please fix the lint issue:
```
hack/tools/lifted-gen/lifted-gen.go:5: File is not `goimports`-ed with -local github.com/karmada-io/karmada (goimports)
	"github.com/spf13/pflag"
```

File: hack/tools/lifted-gen/lifted-gen.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/spf13/pflag"
```
Comment (XiShanYongYe-Chang) [inline hack/tools/lifted-gen/lifted-gen.go:5]: Can be import as:
```
go package

third package

karmada package
```

**Comment by tanberBro** [comment]
Modification completed 
/cc @RainbowMango @XiShanYongYe-Chang

**Review by XiShanYongYe-Chang — COMMENTED** [review]
LGTM
Hi @tanberBro, can you help squash your commits into one?
You can run as follows:
```
$ git reset HEAD~2
$ git add .
$ git commit -s -m "<commit message>"
$ git push origin master -f 
```

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Thanks~
/lgtm

**Comment by RainbowMango** [comment]
Just a reminder, maybe the email present in your commit log is not configured in [GitHub](https://github.com/settings/emails), maybe GitHub can't count your contribution.

**Comment by ikaven1024** [comment]
Thanks~
/lgtm
/approve

**Pull request merged**